orttraining/orttraining/python/training/ortmodule.py

```diff
@@ -213,8 +215,11 @@ def backward(ctx, *grad_outputs):
         # Get the module that flattens the output from the original module into a tuple
         self._flattened_output_module = \
             _ortmodule_output_transformation.get_flattened_output_module(self._original_module)
-        sig = signature(self._original_module.forward)
-        self._original_module_input_names = sig.parameters.keys()
+        self._original_module_parameters = signature(self._original_module.forward).parameters.values()
+        for input_parameter in self._original_module_parameters:
+            if input_parameter.kind == inspect.Parameter.VAR_KEYWORD:
+                warnings.warn("The model's forward method has **kwargs parameter which is not supported.")
```

Comment:
May we put this in a function of its own? We could also add a unit test that hits this ```if``` block?

Reply:
I've added the test.

Not sure this needs to be in a function. I've also added a TODO to remove this when PyTorch ONNX exporter supports **kwargs.